workflows/rnafusion.nf

```diff
@@ -28,7 +28,10 @@ def checkPathParamList = [
 
 
 for (param in checkPathParamList) if ((param) && !params.build_references) file(param, checkIfExists: true)
-if (!("s3" in params.fasta)) {
+if (params.fasta[0,1] == "s3") {
```

Comment:
What's the recommended way to test this? Is there a way to run it with the `-r` flag like for `dev`, or do you have to something more involved?

Reply:
Just run the pipeline with `-r fusionreport` and that should do it :)